js/src/traceable.ts

```diff
@@ -157,7 +161,9 @@ export function traceable<Func extends (...args: any[]) => any>(
                 chunks.push(chunk);
                 yield chunk;
               }
-              await currentRunTree.end({ outputs: chunks });
+              const finalOutputs =
+                aggregator === undefined ? chunks : aggregator(chunks);
```

Comment:
Should we try/catch the aggregator? Woudl probably not want to break the function execution just because someone mis-implemented the function

Reply:
Sure, down for that.